### PR TITLE
added dialog widget

### PR DESCRIPTION
**Pull request by krish-dev-7**
related to issue [#2 ](https://github.com/ACM-VIT/fludget/issues/2)

hope this helps, thanks to make this opportunity to contribute to ACM-VIT ;)

### CODE REVIEW

**Comment by krish-dev-7** [comment]
please give hacktoberfest-accepted

**Comment by DarthBenro008** [comment]
@krish-dev-7 could you please attach some screenshots of your work in the PR?

**Comment by krish-dev-7** [comment]
yeah sure ,
here are the screenshots respective to the pages.

[image: fludget ss1.jpg]

[image: fludget ss2.jpg]






[image: image.png]

**Comment by AryanKhubchandani** [comment]
Hey @krish-dev-7,
Make sure you have attached the screenshots correctly, they are not visible.

**Comment by krish-dev-7** [comment]
@aaryankotharii i have sent it to @DarthBenro008 trough mail....ok will send you too

**Comment by krish-dev-7** [comment]
fludget ss1
fludget ss2
fludget ss4

File: lib/routes/dialogBox.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          child: const Icon(Icons.arrow_back),
+        ),
+        new FlatButton(
+          onPressed: () {
+            Navigator.of(context).pop();
+          },
+          textColor: Theme.of(context).primaryColor,
+          child: const Icon(
+            Icons.done,
+            color: Colors.green,
```
Comment (DarthBenro008) [inline lib/routes/dialogBox.dart:23]: Please give some description about what DialogBox does and a link to its official documentation

**Review by DarthBenro008 — CHANGES_REQUESTED** [review]
@krish-dev-7 Thank you for this PR, but please ensure that you provide a description of the widget and a link to official documentation that people can read more into

**Comment by krish-dev-7** [comment]
I have Added a Dialog Widget named `AlertDialog` inside `ShowDialog`

- In title field we can give title
- in Action field it will take like Row of Widget 
- Finally Wrap the Alert Box with ShowDialog widget with context
- official doccumentations : [AlertDialog](https://api.flutter.dev/flutter/material/AlertDialog-class.html), [ShowDialog](https://api.flutter.dev/flutter/material/showDialog.html)

**Review by krish-dev-7 — COMMENTED** [review]
I have Added a Dialog Widget named `AlertDialog` inside `ShowDialog`

- In title field we can give title
- in Action field it will take like Row of Widget 
- Finally Wrap the Alert Box with ShowDialog widget with context
- official doccumentations : [AlertDialog](https://api.flutter.dev/flutter/material/AlertDialog-class.html), [ShowDialog](https://api.flutter.dev/flutter/material/showDialog.html)

**Comment by AryanKhubchandani** [comment]
@krish-dev-7 thankyou for you contibutiion!

**Pull request merged**